sycl/source/detail/stream_impl.cpp

```diff
@@ -6,6 +6,7 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include <CL/sycl/queue.hpp>
```

Comment:
Should all these includes using `""` instead as they are implementation details and not user-facing system includes? What happens if there is already an incompatible SYCL implementation on the system and you are testing this one at the same time without installing it?